code/modules/reagents/chemistry/recipes/toxins.dm

```diff
@@ -119,3 +119,10 @@
 	id = "mimesbane"
 	results = list("mimesbane" = 3)
 	required_reagents = list("radium" = 1, "mutetoxin" = 1, "nothing" = 1)
+
+/datum/chemical_reaction/bonehurtingjuice
+	name = "Bone Hurting Juice"
+	id = "bonehurtingjuice"
+	results = list("bonehurtingjuice" = 5)
+	required_reagents = list(/datum/reagent/toxin/mutagen = 1, /datum/reagent/toxin/itching_powder = 3, /datum/reagent/consumable/milk = 1)
```

Comment:
required_reagents = list("mutagen" = 1, "itching_powder = 3", "milk" = 1) - This will work

Reply:
reasons to not code when you're tired lmao